<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="11.406">
 <testsuite name="SCT001 - Super Calculator Test" timestamp="2023-05-31T10:45:10" hostname="localhost" time="11.406" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="SCT001 - Super Calculator Test" name="TC001 - Can add two numbers" time="6.027" />
  <testcase classname="SCT001 - Super Calculator Test" name="TC002 - Can subtract two numbers" time="5.363">
   <failure type="toBe" message="Expected &apos;1&apos; to be &apos;10&apos;."><![CDATA[Error: Failed expectation
    at C:\Users\npallado\Desktop\Automation\Practice\Protractor\SuperCalculator\pages\calculatorPage.js:31:52
    at ManagedPromise.invokeCallback_ (C:\Users\npallado\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\npallado\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\npallado\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at C:\Users\npallado\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
    at C:\Users\npallado\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at processTicksAndRejections (node:internal/process/task_queues:95:5)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>